Tests/SourceKitDTests/CrashRecoveryTests.swift

```diff
@@ -42,7 +43,7 @@ fileprivate extension HoverResponse {
 
 final class CrashRecoveryTests: XCTestCase {
   func testSourcekitdCrashRecovery() throws {
-    try XCTSkipUnless(isDarwinHost, "Linux and Windows use in-process sourcekitd")
+    try XCTSkipUnless(Platform.current == .some(.darwin), "Linux and Windows use in-process sourcekitd")
```

Comment:
Nit: Wouldn’t this be nicer as
```suggestion
    try XCTSkipUnless(Platform.current == Platform.darwin, "Linux and Windows use in-process sourcekitd")
```

Or I think `.darwin` might even work as well, but I’m not sure.

Reply:
Ah, right, I think that `.darwin` might work, and that is better.